src/components/ToggleSwitch/ToggleSwitch.module.scss

```diff
@@ -1,15 +1,16 @@
 @use "src/styles/colors";
 
-$switch-height: 1.5rem;
+$switch-height: 1rem;
 $switch-aspect-ratio: 2;
-$switch-padding: 0.25rem;
+$switch-padding: 0.2rem;
 
 .toggleSwitchWrapperOn,
 .toggleSwitchWrapperOff {
```

Comment:
Instead of applying the style to both classes, make a general `.toggleSwitchWrapper`, and the a separate `.on` and `.off`